pages/linux/pveversion.md

```diff
@@ -0,0 +1,12 @@
+# pveversion
+
+> Proxmox VE version info.
```

Comment:
How does this command differ from `pvesh` with respect to functionality? Can you reflect this in the examples shown in the pages?

Reply:
`pvesh` is a shell interface. It's just barebones because I just want this out of the way so that someone else can continue the work.

Reply:
See above.